CHANGES.rst

```diff
@@ -16,6 +16,10 @@ Breaking Changes
   ``False``.
 - ``RequestOptions.auto_parse_qs_csv`` now defaults to ``False`` instead of
   ``True``.
+- ``Request.context_type`` now defaults to a subclass of dict allowing to
+  set attributes on the request context object.
```

Comment:
We should include an additional note about the fact that the mapping and attribute protocols are not magically linked, and so you can not mix-and-match (i.e., we are not trying to be like JavaScript here).

Reply:
Also, we should note that while the context type currently subclasses ``dict`` in order to ease the migration path, the type may be changed to a bare object in a future release (i.e., use of the mapping interface is deprecated as of Falcon 2.0).